edx_lint/cmd/amnesty.py

```diff
@@ -21,7 +21,8 @@
 )
 PYLINT_EXCEPTION_REGEX = re.compile(r"""\s*#\s*pylint:\s+disable=(?P<disables>[^#$]+?)(?=\s*(#|$))""")
 
-PylintError = namedtuple('PylintError', ['filename', 'linenum', 'error_code', 'error_name', 'function', 'error_msg'])
+PylintError = namedtuple("PylintError", ["filename", "linenum", "error_code", "error_name", "function", "error_msg"],)
```

Comment:
Is there a logic to why there's a trailing comma now?

Reply:
It looks like this started with the latest black release (19.10b0), the previous 19.3b0 doesn't do this.  Seems to be an unintended side-effect of https://github.com/psf/black/pull/826 , may be worth filing a bug for.

Reply:
Should we revert the new trailing commas before merging this?  Re-running Black with version 19.3b0 should fix it, not sure if that would make any other undesired changes.